scripts/check_ecr_findings

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -u
```

Comment:
:D

Reply:
I went back and forth on that one, as you can see. Hope you're happy!